### PR TITLE
Updated to chainspec.toml and config for block speed increase improvements.

### PR DESCRIPTION
**Pull request by sacherjj**
Chainspec.toml changes for exp 15 (32s) blocks and seniorage calcs
Reducing deploy_delay from 1min to 15sec for default in config

### CODE REVIEW

**Review by thamps-casp — APPROVED** [review]

**Comment by AlexanderLimonov** [comment]
Looks good

**Review by AlexanderLimonov — APPROVED** [review]

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors try

File: resources/production/config-example.toml [hunk 1]
```diff
@@ -44,7 +44,10 @@ unit_hashes_folder = "/var/lib/casper/casper-node"
 pending_vertex_timeout = '30min'
 
 # If the current era's protocol state has not progressed for this long, shut down.
-# standstill_timeout = '30min'
+standstill_timeout = '30min'
+
+# If after another `standstill_timeout` there still was no progress, shut down.
+shutdown_on_standstill = true
```
Comment (afck) [inline resources/production/config-example.toml]: Sorry, this option was removed actually removed in https://github.com/casper-network/casper-node/pull/1989: The shutdown is now enabled simply by specifying a `standstill_timeout`.

File: resources/production/config-example.toml [hunk 2]
```diff
@@ -44,7 +44,10 @@ unit_hashes_folder = "/var/lib/casper/casper-node"
 pending_vertex_timeout = '30min'
 
 # If the current era's protocol state has not progressed for this long, shut down.
-# standstill_timeout = '30min'
+standstill_timeout = '30min'
+
```
Comment (sacherjj) [inline resources/production/config-example.toml:48]:
```suggestion
```

Comment (sacherjj) [inline resources/production/config-example.toml:48]:
```suggestion
```

**Pull request merged**